src/gap.c

```diff
@@ -1986,7 +1985,7 @@ Obj FuncSHALLOW_SIZE (
 */
 
 Obj FuncTotalMemoryAllocated( Obj self ) {
-    return INTOBJ_INT(SizeAllBags);
+    return ObjInt_Int8(SizeAllBags);
```

Comment:
But `SizeAllBags` is an uint, so this may cause an implicit cast warning later on.

Do you intentionally use `Int8` instead of `UInt8` here? E.g. to catch a hypothetical overflow?

Reply:
There is no ObjInt_UInt8 for some reason. I guess I'll add it.